app/src/main/java/openfoodfacts/github/scrachx/openfood/views/AddProductActivity.java

```diff
@@ -371,10 +371,6 @@ public void onError(Throwable e) {
                         saveProductOffline();
                     }
                 });
-        if(productDetails.get("eating").equals("true")){
```

Comment:
The line 1066... should be removed to and then the method "addProductToList" should be removed as not used